README.md

```diff
@@ -44,15 +44,14 @@ TiDB Insight Scripts
 optional arguments:
   -h, --help            show this help message and exit
   -o OUTPUT, --output OUTPUT
-                        The dir to store output data of TiDB Insight, any
-                        existing file will be overwritten without futher
+                        The directory to store output data of TiDB Insight,
+                        any existing file will be overwritten without futher
                         confirmation.
-  -p, --perf            Collect trace info with perf. Default is disabled.
-  --pid PID             PID of process to run perf on, if '-p/--perf' is not
-                        set, this value will be ignored and would not take any
-                        effection. Multiple PIDs can be set by using more than
-                        one --pid args. Default is None and means the whole
-                        system.
+  -p, --perf            Collect trace info using perf. Disabled by default.
+  --pid PID             PID of process to run perf on, if `-p/--perf` is not
```

Comment:
PID of process to run perf on. **If** -p/--perf is not set, this value will not take effect. Multiple PIDs can be set by using more than one --pid argument. `None` by default which means the whole system.